src/calibrationlib/calibration_table_wrapper.cpp

```diff
@@ -0,0 +1,22 @@
+#include "calibration_table_wrapper.hpp"
+#include <storage/base_encoded_segment.hpp>
+
+namespace opossum {
+CalibrationTableWrapper::CalibrationTableWrapper(
+    const std::shared_ptr<Table> table, const std::string& table_name,
+    const std::vector<ColumnDataDistribution> column_data_distribution_collection)
+    : _table(table), _name(table_name), _column_data_distribution_collection(column_data_distribution_collection) {
+  assert(table->column_count() == column_data_distribution_collection.size());
+}
+
+CalibrationTableWrapper::CalibrationTableWrapper(const std::shared_ptr<Table> table, const std::string& table_name)
+    : _table(table), _name(table_name), _column_data_distribution_collection() {}
+
+const std::shared_ptr<Table> CalibrationTableWrapper::get_table() const { return _table; }
+
+const std::string& CalibrationTableWrapper::get_name() const { return _name; }
+
+const ColumnDataDistribution CalibrationTableWrapper::get_column_data_distribution(ColumnID id) const {
```

Comment:
Looks like a boiler plate class. Wouldn't a method-free struct suffice?

Reply:
I think, if we drop the [assert above](https://github.com/hyrise/hyrise/pull/2009#discussion_r405025220), yes.